lib/github-auth.js

```diff
@@ -110,6 +113,7 @@ function setRoutes(server) {
       }, 10000)
     }
     addGithubToken(data.token)
+    emitter.emit('token-added', data.token)
```

Comment:
Yea… and the next iteration of this, with a redis backend, will use that. I believe Redis can do inserts and deletes instead of rewriting the whole thing.